tests/test_Histogram.py

```diff
@@ -102,7 +102,18 @@ def test_add_value_single_number_to_multiple_histograms():
             [[0, 0, 0, 0, 1, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 1, 0, 0, 0, 0]]
         ),
     )
-
+    
+def test_add_value_out_of_range():
+    # Test adding values outside the histogram range
+    hist = Histogram((0, 10, 10))
+    histogram_original = hist.histograms_.copy() 
```

Comment:
This looks already good. However I would suggest to fill the Histogram with some data (plus some outliers) and add a second histogram with some different data (plus outliers) as well. Like this we also guarantee that it works in a more general case.

Alternatively, we can leave the test as it is and add my description as an other part of the test.